### DISCUSSION TITLE
Add Tooltip to any div element during runtime

### DISCUSSION THREAD

**Discussion by 1Map** [primefaces/community#433] [PrimeVue]
How can I add a PrimeVue Tooltip to a div during runtime.  I get the element from document.getElementById, but after that I need to add a PrimeVue tooltip to that element.

**Comment by ray007** [comment]
That sound quite similar to what I need: show a tooltip for table-cell content in a `DataTable` or `TreeTable`, that's too wide for its column.
From what I've read so far, we need to do this manually with an `OverlayPanel`.
